app/assets/images/payment_icons/gmo-postpay.svg

```diff
@@ -0,0 +1,22 @@
+<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" aria-labelledby="pi-gmo-postpay">
+  <title id="pi-gmo-postpay">GMO後払い</title>

```

Comment:
We need to make the `title` here and the `label` in `payment_icons.yml` the same value.